dev/io.openliberty.jaxws.config_fat/fat/src/io/openliberty/jaxws/config/fat/WebServiceBndServiceRefPropertiesConfigTest.java

```diff
@@ -0,0 +1,174 @@
+        if (server != null && server.isStarted()) {
+            server.stopServer();
+        }
+    }
+    
+    /**
+     * This tests configuring the enableLoggingInOutInterceptor=true in the server.xml file. This tests invokes the test servlet, which uses a @WebServiceRef service client
+     * to send a simple soap request to the SimpleEcho Web Service. 
+     * 
+     * If the configuration is properly applied, the inbound/outbound SOAP Request/Responses will show up in the logs with 
+     * the "REQ_OUT" "RESP_IN". The test verifies expected messages exist in the logs. 
+     * 
+     * 
+     * The test passes serviceRef="server" to tell the SimpleStubClientServlet to use the @WebServiceRef client that's bound to the <service-ref> in the server.xml file
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testEnableLoggingInOutInterceptorServer() throws Exception {
+
+
+    	   String serviceRef = "server";
+            String response = invokeService(serviceRef); // Call the SimpleStubClientServlet invoke the SimpleService Web Service
+            assertTrue("Expected response to contain Pass but contained: " + response, response.contains("Pass"));
+            
+            // Verify message.log contains the SOAP Message logging
+            List<String> dumpInMessages = server.findStringsInLogs("REQ_OUT");
+            List<String> dumpOutMessages = server.findStringsInLogs("RESP_IN");
+            assertTrue("Can't find inBoundMessage, the return inboundmessage is: " + dumpInMessages.toString(), !dumpInMessages.isEmpty());
+            assertTrue("Can't find outBoundMessage, the return outboundmessage is: " + dumpOutMessages.toString(), !dumpOutMessages.isEmpty());
+    }
+    
+    /**
+     * This tests configuring the enableLoggingInOutInterceptor=true in the ibm-ws-bnd.xml file. This tests invokes the test servlet, which uses a @WebServiceRef service client
+     * to send a simple soap request to the SimpleEcho Web Service. 
+     * 
+     * If the configuration is properly applied, the inbound/outbound SOAP Request/Responses will show up in the logs with 
+     * the "REQ_OUT" "RESP_IN". The test verifies expected messages exist in the logs. 
+     * 
+     * The test passes serviceRef="bnd" to tell the SimpleStubClientServlet to use the @WebServiceRef client that's bound to the <service-ref> in the ibm-ws-bnd.xml file
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testEnableLoggingInOutInterceptorIbmWsBnd() throws Exception {
+
+        	String serviceRef = "bnd";
+            String response = invokeService(serviceRef); // Call the SimpleStubClientServlet invoke the SimpleService Web Service
+            assertTrue("Expected response to contain Pass but contained: " + response, response.contains("Pass"));
+            
+
+            // Verify message.log contains the SOAP Message logging
+            List<String> dumpInMessages = server.findStringsInLogs("REQ_OUT");
```

Comment:
Should this be other way around? InMesages (REQ_OUT) and OutMessages (REQ_IN)?

Reply:
This is how the original test was written, I was just keeping the order the same.